python/BUILDING.md

```diff
@@ -3,20 +3,37 @@
 This document describes how to build and install Ice for Python from source.
 You can also download and install a [binary distribution].
 
-* [Building with Pip](#building-with-pip)
+* [Building with Build Python Package Builder](#building-with-build-python-package-builder)
```

Comment:
I google "build python package builder" and there is no hit. So it's not a well-known name/title.

I would say:
Building Python Packages with build

And include a link to https://packaging.python.org/en/latest/key_projects/#build ?